portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,24 @@ function openTab(evt, tabName) {
     evt.currentTarget.classList.add("active");
 }
 
+
+
+function hideButton(id) {
+    var x = document.getElementById(id);
```

Comment:
use const whenever the variable wont change for the duration of the function

Reply:
Would it be the best practice to change the variables to const in the function openTab() as well?

Reply:
Yep! Good catch

Reply:
Got it! Fixed it and the other var/const mix ups in the latest commit